server/src/test/scala/org/http4s/server/middleware/TimeoutSpec.scala

```diff
@@ -48,12 +48,7 @@ class TimeoutSpec extends Http4sSpec {
       var clean = new AtomicBoolean(false)
       val service = HttpService {
         case _ =>
-          implicit val A = Task.asyncInstance(Strategy.sequential)
-          (sleep_(200.milliseconds) ++
-            eval(NoContent()) ++
-            eval_(Task.delay(clean.set(true))))
-            .runLast
-            .map(_.getOrElse(throw new AssertionError("Should have emitted NoContent")))
+          NoContent().schedule(2.seconds) <* Task.delay(clean.set(true))
```

Comment:
It's, like, `*>`, but not. :trollface:  I'll put it in a for comprehension for legibility.